96:07:53.846397 USBPROXY Log continuation - Log started 2023-03-19T09:03:14.240957300Z
96:07:53.887059 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&338c292e&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 6
96:07:53.887277 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&338c292e&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 10
96:07:53.887603 USBPROXY usbLibGetDevices: Found 2 USB devices, 0 captured
97:42:48.997513 USBPROXY usbLibGetDevices: Starting USB device enumeration
97:42:49.031558 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&338c292e&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 6
97:42:49.032212 USBPROXY usbLibGetDevices: Found 2 USB devices, 0 captured
97:43:51.125299 USBPROXY usbLibGetDevices: Starting USB device enumeration
97:43:51.156961 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&338c292e&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 6
97:43:51.157063 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&338c292e&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 10
97:43:51.157254 USBPROXY usbLibGetDevices: Found 2 USB devices, 0 captured
97:44:42.439570 dns-monitor HostDnsMonitor: updating information
97:44:42.439634 dns-monitor HostDnsMonitor: unchanged
98:11:11.225583 USBPROXY usbLibGetDevices: Starting USB device enumeration
98:11:11.226043 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&338c292e&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 6
98:11:11.226494 USBPROXY usbLibGetDevices: Found 2 USB devices, 0 captured
98:11:56.462517 USBPROXY usbLibGetDevices: Starting USB device enumeration
98:11:56.494925 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&338c292e&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 6
98:11:56.495118 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&338c292e&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 10
98:11:56.495366 USBPROXY usbLibGetDevices: Found 2 USB devices, 0 captured
98:23:06.035620 USBPROXY usbLibGetDevices: Starting USB device enumeration
98:23:06.051375 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&338c292e&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 6
98:23:06.052236 USBPROXY usbLibGetDevices: Found 2 USB devices, 0 captured
98:28:56.828086 dns-monitor HostDnsMonitor: updating information
98:28:56.828124 dns-monitor HostDnsMonitor: unchanged
98:34:59.563923 USBPROXY usbLibGetDevices: Starting USB device enumeration
98:34:59.595652 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&338c292e&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 6
98:34:59.595735 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&338c292e&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 10
98:34:59.595872 USBPROXY usbLibGetDevices: Found 2 USB devices, 0 captured
98:37:14.794304 main     VirtualBox: object deletion starts
98:37:14.795405 main     ERROR [COM]: aRC=VBOX_E_OBJECT_IN_USE (0x80bb000c) aIID={ad47ad09-787b-44ab-b343-a082a3f2dfb1} aComponent={MediumWrap} aText={Medium 'C:\Users\zaidk\VirtualBox VMs\Ubuntu 22.04\Ubuntu 22.04.vdi' cannot be closed because it is still attached to 1 virtual machines}, preserve=false aResultDetail=0
98:37:14.807739 main     HostDnsMonitor: shutting down ...
98:37:14.807872 main     HostDnsMonitor: shut down
98:37:14.813096 Watcher  ERROR [COM]: aRC=E_ACCESSDENIED (0x80070005) aIID={7682d5eb-f00e-44f1-8ca2-99d08b1cd607} aComponent={VirtualBoxWrap} aText={The object is not ready}, preserve=false aResultDetail=0
98:37:14.814155 main     VirtualBox: object deleted
